src/components/App/App.js

```diff
@@ -7,7 +7,7 @@ import "react-loader-spinner/dist/loader/css/react-spinner-loader.css"
 
 import "./App.css"
 
-import { LandingPage } from "../LandingPage/LandingPage"
+import { LandingPage } from "../LandingPage/LandingPage.js"
```

Comment:
Hmm wonder why this was necessary to add?

Reply:
Yeah, I'm not sure it wasn't compiling without it for some reason.